test/unit_test/static/quantity_kind_test.cpp

```diff
@@ -32,7 +34,10 @@
 #include <units/quantity_kind.h>
 #include <units/quantity_point.h>
 #include <units/unit.h>
-#include <cassert>
+#else
+import mp_units;
+import <chrono>;
+#endif
 #include <functional>
 #include <limits>
 #include <type_traits>
```

Comment:
Why only `<chrono>` is imported while other STD headers are `#include`d?

Reply:
No particular reason. This highlights that `chrono.h` guarantees `<chrono>` is `#include`d, whereas `mp_units` doesn't export `<chrono>`. Do you think it should?

Reply:
Aren't `#include`s also not exported? It would be nice to be consistent. Also is there any difference in preprocessor parsing overhead between `import <header>` and `#include <header>` or is it equivalent?

Reply:
> Aren't `#include`s also not exported?

That's right.



> Also is there any difference in preprocessor parsing overhead between `import <header>` and `#include <header>` or is it equivalent?

It depends on the implementation. Clang seems to translate `#include`s to `import`. GCC doesn't, and attempting to include standard headers while importing `mp_units` would break it due to declarations being both reachable from `mp_units` (because in the current state, `#include`s happen outside the GFM, but that's OK because Clang translates them to `import`s) as well as from where they were `#include`d.

As for the parsing overhead, I don't have a direct answer. But on my machine, compiling a module that just exports all importable C++ standard library headers required by `mp_units` takes like 0.25s, whereas compiling the `mp_units.core` that imports such a module takes less than 1s, so it's negligible.

Reply:
For now, I `export import std;` in `module mp_units;`.